src/main/java/com/flipkart/zjsonpatch/JsonDiff.java

```diff
@@ -50,16 +45,16 @@ public static JsonNode asJson(final JsonNode source, final JsonNode target, Enum
         generateDiffs(diffs, path, source, target);
 
         if (!flags.contains(DiffFlags.OMIT_MOVE_OPERATION)) {
-          /*
-         * Merging remove & add to move operation
-         */
+            /*
```

Comment:
Why multi-line comments? Just makes everything more difficult...

Reply:
Agreed, will make it Single line comment